libzmix/bulletproofs_amcl/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
We shouldn't need two LICENSE files. This should be a component of zmix and thus be under the same license.

Reply:
OK